node_modules/col-activities/lib/notifications/weekly.js

```diff
@@ -514,41 +522,75 @@ var UserTotals = function() {
     'pointsReceived': 0
   };
 
+  // Define a method to increment activity counts.
+  var activitiesDictionary = {
+    'get_asset_comment': 'commentsReceived',
+    'get_asset_comment_reply': 'commentsReceived',
+    'get_like': 'likesReceived'
+  };
+
+  that.incrementActivities = getActivitiesIncrementer(that, activitiesDictionary);
+
+  // Define a method to increment point totals.
+  var pointsDictionary = {
```

Comment:
Is this dictionary the same as the one in the previous function? If so, it can probably be embedded in the `getPointsIncrementer` function